--- interface.c	2012-03-03 14:04:23.934100265 +0100
+++ interface.c	2012-03-03 14:22:17.808975725 +0100
@@ -26,6 +26,8 @@
 #define GLADE_HOOKUP_OBJECT_NO_REF(component,widget,name) \
   g_object_set_data (G_OBJECT (component), name, widget)
 
+#include "gtkmultilinetabnotebook.c"
+
 GtkWidget*
 create_window1 (void)
 {
@@ -303,6 +305,7 @@
   GtkWidget *notebook_info_label_scribble;
   GtkWidget *hbox1;
   GtkWidget *statusbar;
+  GtkWidget *multiline_tab_notebook;
   GtkAccelGroup *accel_group;
   GtkTooltips *tooltips;
 
@@ -1371,9 +1374,9 @@
   gtk_notebook_set_tab_label (GTK_NOTEBOOK (notebook3), gtk_notebook_get_nth_page (GTK_NOTEBOOK (notebook3), 1), label136);
 
   notebook1 = gtk_notebook_new ();
-  gtk_widget_show (notebook1);
-  gtk_paned_pack2 (GTK_PANED (hpaned1), notebook1, TRUE, TRUE);
-  gtk_notebook_set_scrollable (GTK_NOTEBOOK (notebook1), TRUE);
+  multiline_tab_notebook = gtk_multiline_tab_notebook_new (GTK_NOTEBOOK (notebook1));
+  gtk_widget_show_all (multiline_tab_notebook);
+  gtk_paned_pack2 (GTK_PANED (hpaned1), multiline_tab_notebook, TRUE, TRUE);
 
   scrolledwindow1 = gtk_scrolled_window_new (NULL, NULL);
   gtk_widget_show (scrolledwindow1);
