--- main.c	2012-03-05 09:07:28.564748666 +0100
+++ main.c	2012-03-03 20:14:51.352859923 +0100
@@ -202,12 +202,12 @@
 	}
 
 	/* set the tab placements of the notebooks */
-	gtk_notebook_set_tab_pos(GTK_NOTEBOOK(main_widgets.notebook), interface_prefs.tab_pos_editor);
+	gtk_notebook_set_tab_pos(GTK_NOTEBOOK(main_widgets.notebook), GTK_POS_TOP /* interface_prefs.tab_pos_editor */);
 	gtk_notebook_set_tab_pos(GTK_NOTEBOOK(msgwindow.notebook), interface_prefs.tab_pos_msgwin);
 	gtk_notebook_set_tab_pos(GTK_NOTEBOOK(main_widgets.sidebar_notebook), interface_prefs.tab_pos_sidebar);
 
 	/* whether to show notebook tabs or not */
-	gtk_notebook_set_show_tabs(GTK_NOTEBOOK(main_widgets.notebook), interface_prefs.show_notebook_tabs);
+	gtk_notebook_set_show_tabs(GTK_NOTEBOOK(main_widgets.notebook), FALSE /* interface_prefs.show_notebook_tabs */);
 
 #ifdef HAVE_VTE
 	if (! vte_info.have_vte)
